foundation-extensions/mobile-core/configuration/configuration-api-reference.md

```diff
@@ -255,14 +255,14 @@ MobileCore.configureWithFileInPath("absolute/path/to/exampleJSONfile.json");
 
 #### Example
 
-**Objective-C**
+#### Objective-C
```

Comment:
Formatting - this applies to all occurrences: We usually use h3 for the programming language and bold ** for Example and Syntax
### Objective-C
**Syntax**
```objectivec
your code
```

**Example**
```
your code
```

Reply:
Looks like I was referencing a page that wasn't formatted this way! 
Realized that these docs aren't strictly structured in
_Language ,Syntax, Example_ format  (most of them do not have syntax). Fetching syntaxes is taking longer I expected! Should have a change soon.